src/components/Garden/ProposalDetail/ProposalDetail.js

```diff
@@ -171,143 +177,159 @@ function ProposalDetail({
                   css={`
                     display: grid;
                     grid-template-rows: auto;
-                    grid-row-gap: ${7 * GU}px;
+                    grid-row-gap: ${(isAbstainProposal ? 2 : 7) * GU}px;
                   `}
                 >
                   <div>
-                    <ProposalHeader proposal={proposal} />
-                    <h1
-                      css={`
-                        ${textStyle('title2')};
-                      `}
-                    >
-                      {name}
-                    </h1>
+                    {isAbstainProposal ? (
+                      <AbstainCardHeader proposal={proposal} />
+                    ) : (
+                      <ProposalHeader proposal={proposal} />
+                    )}
```

Comment:
I think for the detail card we can move the hint into a text box, and expand a bit more on the arguments. And maybe put it on a warning text box. Here is a mock of how I think would be great the detail card to look:
<img width="1205" alt="Screen Shot 2022-01-22 at 9 56 32 PM" src="https://user-images.githubusercontent.com/9082013/150660544-e1ec0e42-0851-43b7-a00d-aa7c2ff5e337.png">

Reply:
The long version of the explanation:
```
        This is a special kind of suggestion proposal that is always available.
        It serves the purpose of regulating the community&apos;s expenditure by
        increasing the amount of support required for all other funding
        proposals to pass. 
        
        So if you think that the community is spending money
        unproductively supporting the abstain proposal, or creating a new
        signaling proposal that makes the case for how you think the community
        should actually be allocating resources have a meaningful influence on
        the behavior of the system.
 ```

Reply:
WDYT @0xGabi 

![Captura de ecrã 2022-01-23, às 12 22 08](https://user-images.githubusercontent.com/13184907/150678163-2877610d-45f0-44e6-a82e-d8d3474eb461.png)

Reply:
Looking great. The only thing I am not convinced of is about having the info box. Let's keep the same text we were using on the other proposals 👍

Reply:
Like this? @0xGabi 

<img width="1439" alt="Captura de ecrã 2022-01-24, às 23 38 43" src="https://user-images.githubusercontent.com/13184907/150883319-6c9de182-0ce7-49a0-90e8-abb777978292.png">

Reply:
Yes, great 🙌